packages/smooth_app/lib/bottom_sheet_views/user_contribution_view.dart

```diff
@@ -0,0 +1,294 @@
+                                                  .contribute_contribute_toBeCompleted,
+                                              style: Theme.of(context)
+                                                  .textTheme
+                                                  .bodyText1,
+                                            ),
+                                          ),
+                                          FlatButton(
+                                            onPressed: () => launcher.launchURL(
+                                                context,
+                                                'https://wiki.openfoodfacts.org/Contribution_missions',
+                                                false),
+                                            child: Text(
+                                              S
+                                                  .of(context)
+                                                  .contribute_contribute_contributionMissions,
+                                              style: const TextStyle(
+                                                  color: Colors.blue),
+                                            ),
+                                          ),
+                                          FlatButton(
+                                            onPressed: () => launcher.launchURL(
+                                                context,
+                                                'https://wiki.openfoodfacts.org/Quality',
+                                                false),
+                                            child: Text(
+                                              S
+                                                  .of(context)
+                                                  .contribute_contribute_qualityIssues,
+                                              style: const TextStyle(
+                                                  color: Colors.blue),
+                                            ),
+                                          ),
+                                        ],
+                                      ),
+                                      actions: <SmoothSimpleButton>[
+                                        SmoothSimpleButton(
+                                          onPressed: () {
+                                            Navigator.of(context,
+                                                    rootNavigator: true)
+                                                .pop('dialog');
+                                          },
+                                          text: '${S.of(context).okay}',
+                                          width: 100,
+                                        ),
+                                      ],
+                                    );
+                                  })),
+
+                          //Develope
+                          SmoothListTile(
```

Comment:
Each SmoothListTile's onPressed could be written in a separate function for extra readability since building a dialog is pretty wordy but that's a detail :wink:

Reply:
I'm young and naive, done